src/arvdevice.c

```diff
@@ -393,6 +393,35 @@ _get_feature (ArvDevice *device, GType node_type, const char *feature, GError **
 	return node;
 }
 
+/**
+ * arv_device_get_representation:
+ * @device: a #ArvDevice
+ * @feature: feature name
+ * @error: a #GError placeholder, %NULL to ignore
+ *
+ * Return: enum ArvGcRepresentation, -1 if not available.
+ *
+ * Since: 
+ */
+
+ArvGcRepresentation
+arv_device_get_representation (ArvDevice *device, const char *feature, GError **error)
+{
+	ArvGcNode* node;
+
+	g_return_val_if_fail (ARV_IS_DEVICE (device), FALSE);
+	g_return_val_if_fail (feature != NULL, FALSE);
```

Comment:
Type mismatch, return REPRESENTATION_UNDEFINED instead, and update the doc accordingly

Reply:
updated in dbf13af